TriblerGUI/widgets/trustpage/js/parser.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Parse a JSON string into the defined format. This file explains what JSON is expected and how the resulting
+ * javascript objects are constructed. The type definitions can be used for type hinting.
+ *
+ * The expected JSON format:
+ *
+ * {
+ *       "focus_node": "xyz",
+ *       "neighbor_level": 1,
+ *       "nodes": [{
+ *           "public_key": "xyz",
+ *           "total_up": 12736457,
+ *           "total_down": 1827364,
+ *           "page_rank": 0.5
+ *       }, ...],
+ *       "edges": [{
+ *           "from": "xyz",
+ *           "to": "xyz_n1",
+ *           "amount": 12384
+ *       }, ...]
+ * }
+ *
+ * This will be converted into the following object:
+ */
+
+/**
+ * Convert a JSON string into a GraphResponseData Object
+ * @param {String} json - The JSON string assumed to have the defined format
+ * @returns {GraphResponseData} - The response object
+ */
+function parse(json){
+    return JSON.parse(json);
+}
```

Comment:
Yes I was thinking about this. This file contains mainly an explanation about the expected JSON format and resulting Javascript Object. This may not be the best way to do this though, but I would like to keep it out of the data_processor.js so that file does not have to care about the JSON provided by the API, only about the GraphResponseData.

Any ideas?